webapp/channels/src/utils/test_helper.ts

```diff
@@ -268,6 +268,28 @@ export class TestHelper {
         return Object.assign({}, defaultIncomingWebhook, override);
     }
 
+    public static getOutgoingWebhookMock(override: Partial<OutgoingWebhook> = {}): OutgoingWebhook {
+        const defaultOutgoingWebhook: OutgoingWebhook = {
+            id: 'id',
+            token: 'hook_token',
+            create_at: 1,
+            update_at: 1,
+            delete_at: 1,
```

Comment:
lets assign these to 0 for starters, it could be overridden if user wants in a test